content/cybersecurity/concepts/nmap/terms/os-detection/os-detection.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: 'OS Detection'
+Description: 'One of Nmap's best-known features is remote OS detection using TCP/IP stack fingerprinting.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+SeeAlso:
+  - 'aggressive-scan'
+---
+
+One of Nmap's best known features is remote **OS Detection** using TCP/IP stack fingerprinting. With **OS Detection**, Nmap sends a series of TCP and UDP packets to the specified host and examine the reponces. Nmap then compares the results to its *nmap-os-db* database of more than 2,600 known OS fingerprints and prints out the OS details if there is a match.  
+Each fingerprint includes a freeform textual description of the OS, and a classification which provides the vendor name (e.g. Sun), underlying OS (e.g. Solaris), OS generation (e.g. 10), and device type (general purpose, router, switch, game console, etc). Most fingerprints also have a Common Platform Enumeration (CPE) representation, like `cpe:/o:linux:linux_kernel:2.6.`
+
+Note that *root* privilages are required to properly perform the scan.
+
+
+**OS Detection** is enabled with:
+
+- Enable OS detection (`-O`)
+
+**OS Detection** is controlled with the following options (be sure to include with `-O`):
+- Limit **OS detection** to promising targets (`--osscan-limit`)
+    - This can save substantial time by only scanning targets that have a critera that makes **OS Detection** more effective, which is if there is at least one open and one closed TCP port are found. Nmap will now attempt **OS Detection** against hosts which do not meet that criteria.
```

Comment:
```suggestion
    - This can save substantial time by only scanning targets that have a criteria that makes OS detection more effective, which is if there is at least one open and one closed TCP port are found. Nmap will not attempt OS detection against hosts which do not meet that criteria.
```